### DISCUSSION TITLE
Why is swap-on ignored by default?

### DISCUSSION THREAD

**Discussion by turowicz** [rancher/rke2#1925] [Q&A]
Similar to microk8s, RKE2 starts kubelet with `--fail-swap-on=false`. Is there a reason for this? Kubernetes documentation says that swap is not yet supported.

**Comment by turowicz** [comment]
Perhaps in the quick install docs it should explain why swap is undesirable for k8s.

**Comment by brandond** [comment]
This is a customization to allow K3s and RKE2 on nodes that it would not otherwise be able to even start on. We don't believe that the Kubelet should just crash on startup if users want to run it on a node with swap enabled.
